build.wake

```diff
@@ -132,22 +173,29 @@ def tarball Unit =
     findFailFn getPathResult (tarball, changelog, Nil)
 
 export def static _: Result Path Error =
-    def filesResult = doInstall "tmp" "static"
-    def releaseResult = buildAs Unit
-    def timeResult = buildOn Unit
-    require Pass release = releaseResult
-    require Pass time = timeResult
-    require Pass files = filesResult
+    require Pass files =
+        doInstall "tmp" "static"
+    require Pass release =
+        buildAs Unit
+    require Pass time =
+        buildOn Unit
     def cmd =
-        which "tar", "--mtime={time}", "--transform=s@^tmp/@wake-{release}/@",
-        "--owner=0", "--group=0", "--numeric-owner", "-cJf",
-        "wake-static_{release}.tar.xz", map getPathName files | sortBy (_<*_)
+        which "tar",
+        "--mtime={time}",
+        "--transform=s@^tmp/@wake-{release}/@",
+        "--owner=0",
+        "--group=0",
+        "--numeric-owner",
+        "-cJf",
+        "wake-static_{release}.tar.xz",
+        map getPathName files | sortBy (_<*_)
```

Comment:
I don't know what I was thinking with this change. I'll still probably put the map call on a separate line, and maybe have the which "tar" separate as well, but the rest should probably stay together.